src/test/scala/se/nimsa/dcm4che/streams/DicomData.scala

```diff
@@ -96,10 +96,10 @@ object DicomData {
         case chunk: DicomValueChunk => chunk.bytes == bytes
       }
 
-    def expectItem() = probe
+    def expectItem(index: Int) = probe
```

Comment:
Tests now assert both that we are seeing an item in the stream and that the item index is as expected